AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessor.swift

```diff
@@ -227,6 +229,8 @@ final class ItemListProcessor: StateProcessor<ItemListState, ItemListAction, Ite
     /// Stream the items list.
     private func streamItemList() async {
         do {
+            await determinePasswordManagerSyncVisibility()
+
```

Comment:
🎨  IMO this is not strictly part of streaming the item list, so I think it'd be better to be directly in `appeared` instead of here.
```swift
        case .appeared:
            await determinePasswordManagerSyncVisibility()
            await streamItemList()
```